pkg/apis/config/v1beta1/defaults.go

```diff
@@ -37,21 +36,27 @@ var (
 	}
 
 	defaultKubeConfigPath string = "/etc/kubernetes/scheduler.conf"
+	defaultKubeMaster     string = "127.0.0.1:32768"
 )
 
-func SetDefaults_CoschedulingArgs(obj *CoschedulingArgs) {
+// SetDefaultsCoschedulingArgs sets the default parameters for Coscheduling plugin.
+func SetDefaultsCoschedulingArgs(obj *CoschedulingArgs) {
 	if obj.PermitWaitingTimeSeconds == nil {
 		obj.PermitWaitingTimeSeconds = &defaultPermitWaitingTimeSeconds
 	}
-	if obj.PodGroupGCIntervalSeconds == nil {
-		obj.PodGroupGCIntervalSeconds = &defaultPodGroupGCIntervalSeconds
+	if obj.DeniedPGExpirationTimeSeconds == nil {
+		obj.DeniedPGExpirationTimeSeconds = &defaultDeniedPGExpirationTimeSeconds
 	}
-	if obj.PodGroupExpirationTimeSeconds == nil {
-		obj.PodGroupExpirationTimeSeconds = &defaultPodGroupExpirationTimeSeconds
+	if obj.KubeConfigPath == nil {
+		obj.KubeConfigPath = &defaultKubeConfigPath
+	}
+	if obj.KubeMaster == nil {
+		obj.KubeMaster = &defaultKubeMaster
```

Comment:
I not  sure if we need default values here

Reply:
Maybe not. Actually, I don't think we need kubeCongfigPath and kubeMaster. They are defined in kubeconfig.

Reply:
I  also think we can remove this when framework supporting directly getting kubeClient form framework

Reply:
@cwdsuzhou  I removed default `KubeConfigPath` and `KubeMaster`.  Hopefully we can get the two parameters from the existing configurations or command args.  Submitted an issue https://github.com/kubernetes/kubernetes/issues/96427.